examples/07-EMIT/interference_gui.py

```diff
@@ -0,0 +1,509 @@
+import sys
+from PySide2 import QtWidgets, QtUiTools, QtGui
+from pyaedt.emit_core.emit_constants import InterfererType, ResultType, TxRxMode
+from pyaedt import Emit
+import pyaedt
+from openpyxl.styles import PatternFill
+import openpyxl
+import os
+import pyaedt.generic.constants as consts
+
+sys.path.append('C:\\Program Files\\AnsysEM\\v232\\Win64\\Delcross')
```

Comment:
I added the version check, I also launched the emit app earlier and used emitapp._desktop_install_dir + "/Delcross" instead of the hard coded path and it is working as expected